docs/fx-core/aad-help.md

```diff
@@ -150,4 +150,14 @@ Root cause of this error is that toolkit can not find Azure AD app with the same
 
 If you can find your Azure AD app, please check your network status and try again.
 
-If you can not find your Azure AD app, please check whether you logged in with the correct account. You can also remove objectId from env.default.json file and then try again.
\ No newline at end of file
+If you can not find your Azure AD app, please check whether you logged in with the correct account. You can also remove objectId from env.default.json file and then try again.
+
+## aad.Compliance
+To improve the security of app registration,please refer to [Best Practices for app registration](
+https://docs.microsoft.com/en-us/azure/active-directory/develop/security-best-practices-for-app-registration) to get detail. The following are several improvements.
+### Use certificate credentials
```

Comment:
Why do we just highlight the *credentials* section? Is it because this is the only part that's relevant to Teams Toolkit? According to the aforementioned reference, there are several areas that worth users' attention:

- Redirect URI
- Implicit grant flow for access token
- Credentials
- AppId URI
- Application ownership

Reply:
If TTK indeed covers more areas, say "Implicit grant flow for access token" and get the alerts, we should probably put that in the recommendations too.

Reply:
The AAD app integration assistant is a general compliance check. Currently, it does not have proper input for user to describe the usage of the application id accurately. 
Other areas like Redirect URI/AppId URI are actually used in the TTK. Implicit grant flow for access token is used for other auth scenarios. They are used in compliance yet.
About the Application ownership, I haven't got it in AAD app integration assistant. We can sync it offline.